_config/importantNews.yml

```diff
@@ -9,6 +9,12 @@
 #
 # Order news by date.
 # Don't remove past news.
+- newsId: 20211129-blog
+  newsDate: 2021-11-29
+  newsTitle: Take a look at jBPM 7.62.0
+  newsContent: jBPM 7.62.0 is out, including bug fixes and exciting new features!
+  newsUrl: https://docs.jbpm.org/7.62.0.Final/jbpm-docs/html_single/#jbpmreleasenotes
```

Comment:
the URL is ok, but the latest news are for 7.52.0.Final, we need to ask jbpm team to fill it @krisv @lazarotti ?

Reply:
@mareknovotny yes I see - the latest news update is  `33.1. jBPM 7.52`